NitroxClient/MonoBehaviours/Gui/Chat/PlayerChatEntry.cs

```diff
@@ -76,9 +78,17 @@ private void SendMessage()
         {
             if (chat != null && chatMessage.Length > 0)
             {
-                chatBroadcaster.SendChatMessage(chatMessage);
-                ChatLogEntry chatLogEntry = new ChatLogEntry("Me", chatMessage, multiplayerSession.PlayerSettings.PlayerColor);
-                chat.WriteChatLogEntry(chatLogEntry);
+                if(chatMessage.Substring(0, 1) == "/")
```

Comment:
You can do `chatMessage[0] == '/'` so you don't have to allocate a new string to compare it.